scripts/test

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
+# -p flag will create a different network for the test container so it won't interfere with the app container
```

Comment:
This is why I was talking about the network: https://docs.docker.com/compose/networking/ 

Note: Your app’s network is given a name based on the “project name”, which is based on the name of the directory it lives in. You can override the project name with either the --project-name flag or the COMPOSE_PROJECT_NAME environment variable.